### PR TITLE
PIM-9750: Add "Troubleshooting guide" section

### PR DESCRIPTION
**Pull request by jmleroux**
Add a "Troubleshooting guide" section and add the first item for PIM-9750

Peek 2021-03-19 18-40

### CODE REVIEW

**Review by fitn — APPROVED** [review]

File: troubleshooting/index.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+Auto_increment reaches max value for pim_catalog_completeness table
+-------------------------------------------------------------------
+
+.. note::
+
+   Impacted versions: PIM <= 5.0
+
+After some time, the completeness table auto_increment identifier could reach the defined maximum.
+We made a fix to to allow for a lot more identifier in PIM-9750 ticket but for aloder database schema,
```
Comment (StevenVAIDIE) [inline troubleshooting/index.rst:12]:
```suggestion
We made a fix to to allow for a lot more identifier in PIM-9750 ticket but for older database schema,
```

**Review by StevenVAIDIE — APPROVED** [review]

File: troubleshooting/index.rst [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+you must apply this fix with the following MySql ALTER command:
+
+.. code-block:: sql
+
+   ALTER TABLE pim_catalog_completeness MODIFY id int AUTO_INCREMENT
```
Comment (Doodoune) [inline troubleshooting/index.rst:17]: Isn't it "bigint" instead of int?

File: troubleshooting/index.rst [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+   Impacted versions: PIM <= 5.0
+
+After some time, the completeness table auto_increment identifier could reach the defined maximum.
+We made a fix to to allow for a lot more identifier in PIM-9750 ticket but for older database schema,
+you must apply this fix with the following MySql ALTER command:
+
+.. code-block:: sql
+
+   ALTER TABLE pim_catalog_completeness MODIFY id int AUTO_INCREMENT
```
Comment (Doodoune) [inline troubleshooting/index.rst:17]:
```suggestion
   ALTER TABLE pim_catalog_completeness MODIFY id bigint AUTO_INCREMENT
```

Comment (Doodoune) [reply troubleshooting/index.rst:17]: Isn't it "bigint" instead of int?

**Review by BitOne — APPROVED** [review]
Thanks for the cleanup and multiple doc fixes!

**Review by Doodoune — APPROVED** [review]

**Pull request merged**